frontend/src/app/main/ui/ds/utilities/swatch.stories.jsx

```diff
@@ -47,52 +48,45 @@ export const WithOpacity = {
 // These stories are disabled because the gradient and the UUID variants cannot be translated from cljs into JS
 // When the repo is updated to use the new version of rumext, these stories should be re-enabled and tested
 //
-// export const LinearGradient = {
-//   args: {
-//     background: {
-//       gradient: {
-//         type: "linear",
-//         startX: 0,
-//         startY: 0,
-//         endX: 1,
-//         endY: 0,
-//         width: 1,
-//         stops: [
-//           {
-//             color: "#fabada",
-//             opacity: 1,
-//             offset: 0,
-//           },
-//           {
-//             color: "#cc0000",
-//             opacity: 0.5,
-//             offset: 1,
-//           },
-//         ],
-//       },
-//     },
-//   },
-// };
-
-// export const Rounded = {
-//   args: {
-//     background: {
-//       id: crypto.randomUUID(),
-//       color: "#7efff5",
-//       opacity: 0.5,
-//     },
-//   },
-// };
+export const LinearGradient = {
+  args: {
+    background: {
+      gradient: {
+        type: helpers.linear,
+        "start-x": 0,
+        "start-y": 0,
+        "end-x": 1,
+        "end-y": 0,
+        width: 1,
+        stops: helpers.stops,
+      },
+    },
+  },
+};
 
-export const Small = {
+export const RadialGradient = {
   args: {
-    size: "small",
+    background: {
+      gradient: {
+        type: helpers.radial,
+        "start-x": 0,
+        "start-y": 0,
+        "end-x": 1,
+        "end-y": 0,
+        width: 1,
+        stops: helpers.stops,
+      },
+    },
   },
 };
 
-export const Active = {
+export const Rounded = {
   args: {
-    active: true,
+    background: {
+      id: helpers.uuid(crypto.randomUUID()),
```

Comment:
I think we can expose directly our own uuid generation function instead invoking crypto here (a very minor issue)